### PR TITLE
Verifications audit

### PR DESCRIPTION
**Pull request by amiedes**
TBD

### CODE REVIEW

File: decidim-core/lib/decidim/attribute_obfuscator.rb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+    def self.name_hint(name)
+      return nil unless name.present?
+      return "#{name.first}#{'*' * (name.length - 2)}#{name.last}" unless name.length > 6
+
+      "#{name[0..2]}#{'*' * (name.length - 6)}#{name[-3..]}"
+    end
+
+    def self.census_attribute_hint(value)
+      value = value.to_s
```
[the remark this answers is not in the document]
Comment (ferblape) [reply decidim-core/lib/decidim/attribute_obfuscator.rb]: Ping

File: decidim-core/lib/decidim/attribute_obfuscator.rb [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    def self.email_hint(full_email)
+      return nil unless full_email.present? && full_email.include?("@")
+
+      segments = full_email.split("@")
+      segment_1 = segments.first
+      segment_2 = segments.second
+
+      segment_1 = if segment_1.length < 4
+                    "#{segment_1.first}#{'*' * (segment_1.length - 1)}"
+                  else
```
[the remark this answers is not in the document]
Comment (amiedes) [reply decidim-core/lib/decidim/attribute_obfuscator.rb]: Does not work in plain Ruby, but it does in rails

File: decidim-core/lib/decidim/attribute_obfuscator.rb (hunk 1 above)
[the remark this answers is not in the document]
Comment (amiedes) [reply decidim-core/lib/decidim/attribute_obfuscator.rb]: This is fixed now:

```
 value = value.to_s

 return nil if value.blank?
```

**Comment by amiedes** [comment]
Closing in favor of https://github.com/PopulateTools/decidim/pull/3, which has the same code but based on a stable branch

**Pull request closed without merging**